webapp/src/index.tsx

```diff
@@ -85,15 +86,15 @@ export default class Plugin {
     userActivityWatch(): void {
         // Listen for new activity to trigger a call to the server
         // Hat tip to the Github and Playbooks plugin
-        let lastActivityTime = Number.MAX_SAFE_INTEGER;
-        const activityTimeout = 60 * 60 * 1000; // 1 hour
+        let nextCheckAfter = Date.now() + Math.max(MINUTE, randomInt(10) * MINUTE);
```

Comment:
I think this creates just ~9 "buckets" across which requests are randomized (1 minute, 2 minutes, ..., 9 minutes). Could we tweak this to randomize across seconds?

```suggestion
        let nextCheckAfter = Date.now() + Math.max(MINUTE, randomInt(10 * MINUTE));
```